v2/schema/context.jsonld

```diff
@@ -0,0 +1,131 @@
+{
+  "@context": {
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "dcterms": "http://purl.org/dc/terms/",
+    "dctypes": "http://purl.org/dc/dcmitype/",
+    "gbl": "http://geoblacklight.org#schema",
```

Comment:
probably should be `http://geoblacklight.org/schema/1.0#` and I should add the `/1.0#` to our JSON-Schema doc